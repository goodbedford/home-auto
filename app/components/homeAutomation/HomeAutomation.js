import React from "react";


const HomeAutomation = (props) => {

  return (
    <div className="home-auto home-auto--md home-auto--sm">
      {props.children}
    </div>
  );
};


export default HomeAutomation;
