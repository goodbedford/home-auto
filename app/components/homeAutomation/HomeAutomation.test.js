/* eslint-disable no-undef */

import React from "react";
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import HomeAutomation from "./HomeAutomation";

describe("<HomeAutomation />", () => {
  it("HomeAutomation snapshot", () => {
    const wrapper = shallow(<HomeAutomation />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
