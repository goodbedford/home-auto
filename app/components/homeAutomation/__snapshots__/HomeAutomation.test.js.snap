// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeAutomation /> HomeAutomation snapshot 1`] = `
<div
  className="home-auto home-auto--md home-auto--sm"
/>
`;
